dlrover/python/master/node/training_node.py

```diff
@@ -127,9 +128,23 @@ def cut_timeout_pending_node_cpu(node: Node):
             _dlrover_context.seconds_to_wait_pending_pod,
             new_cpu,
         )
-        return True
-    else:
-        return False
+        reduced = True
+    original_memory = node.config_resource.memory
+    new_memory = math.ceil(
+        original_memory / _dlrover_context.factor_to_cut_pending_cpu
```

Comment:
Should it be `factor_to_cut_pending_mem`? Also, I think we'd better set a threshold, e.g., 32g. When the original memory is more than the threshold, we conduct resource reduction.

Reply:
Now, we reduce the memory only when `original memory/ factor_to_cut_pending_mem > NodeResourceLimit.MIN_MEMORY` where NodeResourceLimit.MIN_MEMORY= 6GB.